src/components/WwwFrame/TheHeader.vue

```diff
@@ -39,45 +34,35 @@
 						/>
 						<div class="tw-flex-1"></div>
 						<span
-							v-show="hasBasket"
 							@click="$emit('show-basket')"
 							data-testid="header-basket"
-							class="header__button header__basket tw-cursor-pointer tw-inline-flex"
+							class="header__button header__basket tw-cursor-pointer"
 							v-kv-track-event="['TopNav','click-Basket']"
+							:style="isBasketLoading ? {
+								display: 'var(--ui-data-corporate-basket-count-display, inline-flex)',
+							} : {
+								display: hasBasket ? 'inline-flex' : 'none'
+							}"
 						>
 							<span class="tw-bg-secondary tw-rounded-sm tw-py-0.5 tw-px-1 tw-mr-1">
-								{{ basketCount - lcaLoanCount }}
+								<div v-if="isBasketLoading" class="tw-w-1 tw-h-3">
+									<kv-loading-placeholder />
+								</div>
+								<template v-else>
+									{{ basketCount - lcaLoanCount }}
+								</template>
 							</span>
 							Basket
 						</span>
-						<router-link
+						<my-kiva-button
```

Comment:
I think we might want to rename this button, since MyKiva is no longer a portfolio replacement. So maybe this component should be renamed to `PorfolioButton`?

Reply:
I was considering that, but I'm not sure. We've always called this button the "my kiva" button even before the recent MyKiva experience. We have similar naming with the My Kiva dropdown that hovering over this button reveals, and the My Kiva sections of the lend menu.

Reply:
This morning I saw some early mocks for the nav changes coming, and there might be a MyKiva button in the nav. So with that in mind, what are your thoughts?